src/module_writer.rs

```diff
@@ -833,5 +833,14 @@ pub fn write_dist_info(
         )?;
     }
 
+    if !metadata21.license_files.is_empty() {
+        for path in &metadata21.license_files {
+            let filename = path.file_name().with_context(|| {
+                format!("missing file name for license file {}", path.display())
+            })?;
+            writer.add_file(dist_info_dir.join(filename), path)?;
```

Comment:
should you not create a 'license_files' subdirectory like the PEP639 mandates?

https://peps.python.org/pep-0639/#license-files-in-project-formats

>  the .dist-info directory MUST contain a license_files subdirectory which MUST contain the files listed in the License-File fields in the METADATA file at their respective paths relative to the license_files directory.

Reply:
Thanks for the review, admittedly I haven't read through PEP 639 yet so I marked this as a draft.